src/lib/maplibre-util.tsでtscでエラーが発生している

```
src/lib/maplibre-util.ts:54:26 - error TS2352: Conversion of type 'CSSStyleDeclaration' to type 'Record<string, string>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'CSSStyleDeclaration'.

54       DOM.#userSelect = (DOM.#docStyle as Record<string, string>)[
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/lib/maplibre-util.ts:57:8 - error TS2352: Conversion of type 'CSSStyleDeclaration' to type 'Record<string, string>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'CSSStyleDeclaration'.

57       (DOM.#docStyle as Record<string, string>)[DOM.#selectProp] = "none";
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/lib/maplibre-util.ts:63:8 - error TS2352: Conversion of type 'CSSStyleDeclaration' to type 'Record<string, string>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'CSSStyleDeclaration'.

63       (DOM.#docStyle as Record<string, string>)[DOM.#selectProp] =
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/lib/maplibre-util.ts:69:6 - error TS2352: Conversion of type 'CSSStyleDeclaration' to type 'Record<string, string>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'CSSStyleDeclaration'.

69     (el.style as Record<string, string>)[DOM.#transformProp] = value;
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 4 errors in the same file, starting at: src/lib/maplibre-util.ts:54
```

--- a/src/lib/maplibre-util.ts
+++ b/src/lib/maplibre-util.ts
@@ -51,22 +51,23 @@ export class DOM {
 
   static disableDrag(): void {
     if (DOM.#docStyle && DOM.#selectProp) {
-      DOM.#userSelect = (DOM.#docStyle as Record<string, string>)[
+      DOM.#userSelect = (DOM.#docStyle as unknown as Record<string, string>)[
         DOM.#selectProp
       ];
-      (DOM.#docStyle as Record<string, string>)[DOM.#selectProp] = "none";
+      (DOM.#docStyle as unknown as Record<string, string>)[DOM.#selectProp] =
+        "none";
     }
   }
 
   static enableDrag(): void {
     if (DOM.#docStyle && DOM.#selectProp) {
-      (DOM.#docStyle as Record<string, string>)[DOM.#selectProp] =
+      (DOM.#docStyle as unknown as Record<string, string>)[DOM.#selectProp] =
         DOM.#userSelect ?? "";
     }
   }
 
   static setTransform(el: HTMLElement, value: string): void {
-    (el.style as Record<string, string>)[DOM.#transformProp] = value;
+    (el.style as unknown as Record<string, string>)[DOM.#transformProp] = value;
   }
 
   static addEventListener(

